### ISSUE TITLE
Authentication Failed: No server configuration found for issuer:

### ISSUE THREAD

**Issue by itslogan4u** [mitreid-connect/OpenID-Connect-Java-Spring-Server#795]
Hi,

I am getting the following error when I click the login button from the client application. While debugging I found that client failed to form jsonstring from restTemplate object from DynamicServerConfigurationService class. Your help is highly appreciated to resolve this issue.

message Authentication Failed: No server configuration found for issuer: http://10.12.11.207:8080/openid-connect-server-webapp/
description This request requires HTTP authentication.

Thanks,
Loganathan

**Comment by jricher** [comment]
Which server configuration component are you using? Dynamic or hybrid?

Have you set the issuer on the server? This is set in the server-config.xml file.

Is the server's configuration document reachable at:

http://10.12.11.207:8080/openid-connect-server-webapp/.well-known/openid-configuration

? If it's not, the client won't be able to find it.

Can the client reach the above URL on your network? Not your browser, but the client application itself, from wherever it is deployed.

**Comment by itslogan4u** [comment]
Hello,

We have configured it for Dynamic and we set the issuer (http://10.12.11.207:8080/openid-connect-server-webapp/) in server-config file. Server and Client are deployed in same tomcat server(10.12.11.207). 

When I try to launch the URL (http://10.12.11.207:8080/openid-connect-server-webapp/.well-known/openid-configuration) from browser getting the following error.

HTTP Status 500 - Request processing failed; nested exception is org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is javax.persistence.PersistenceException: Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.5.1.v20130918-f2b9fc5): org.eclipse.persistence.exceptions.DatabaseException

org.springframework.web.util.NestedServletException: Request processing failed; nested exception is org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is javax.persistence.PersistenceException: Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.5.1.v20130918-f2b9fc5): org.eclipse.persistence.exceptions.DatabaseException
Internal Exception: org.apache.commons.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (Communications link failure

Last packet sent to the server was 0 ms ago.)
